.github/workflows/scripts/pool.py

```diff
@@ -0,0 +1,25 @@
+from ic.client import Client
+from ic.identity import Identity
+from ic.agent import Agent
+import os
+import sys
+import base64
+from ic.candid import encode, Types
+
+private_key = base64.b64decode(os.environ["DFX_IDENTITY_PREVIEW"]).decode("utf-8")
+pool_id = os.environ["POOL_CANISTER_ID"]
```

Comment:
For maintainability I would appreciate a link to the repo where the pool code is located

Reply:
Good input, adding it.